### DISCUSSION TITLE
Unable to edit DNS records after switch from "Personal Account" to "Free Team"

### DISCUSSION THREAD

**Discussion by snapsnapturtle** [vercel/community#6324] [Help] [answered]
### Summary

Alright, so I have a third party domain set-up in Vercel and the name servers are pointed to the one's from Vercel. About a week ago, I configured a few additional DNS records for that domain in Vercel (before being moved to the "Free Team" tier). Today, these DNS records seem to have disappeared (at least from the UI) and I am also no longer to create new entries. When checking the previously created records with `dig`, they still point to the correct places (which is good).

According to [the announcement](https://vercel.com/changelog/2024-01-account-changes), nothing should have changed, but this has 😔

![image](https://github.com/vercel/community/assets/8927747/114bbf54-7cc4-4c5f-87f8-4eee58d59f10)

### Example

n/a

### Steps to Reproduce

1. Click on profile picture in top right corner
2. Select "Account Settings"
3. On the left sidebar, select "Domains"
4. Click on the domain name in the list
5. In the form to add a new DNS record, type "test" for the name
6. Click on "Add"
7. Receive error message "You don't have permission to create the domain record."
8. Existing domain records are not shown

**Comment by amyegan** [comment, marked as answer]
Hey @snapsnapturtle. Domains that belonged to your personal account before the update should still be listed on the [vercel.com/account/domains](https://vercel.com/account/domains) page. If you [move the domain](https://vercel.com/docs/projects/domains/transfer-your-domain) to your new free team, are you able to modify the DNS records from there?

**Reply by snapsnapturtle** [reply]
Yes, when moving the domain to the new free team, I am able to see and modify the records again. It might be nice to clarify that this step is necessary (or move them automatically with the migration of the accounts).
